### PR TITLE
Added the setup files for the project

### PR DESCRIPTION
**Pull request by drteresavasquez**
## Description
Added the setup files for the project

## Related Issue
Did not have files

## Motivation and Context
Need files to start

## How Can This Be Tested?
`git fetch origin setup-branch`

run the code

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+body {
+    background-color: aquamarine;
```
Comment (drteresavasquez) [inline main.css:2]: This needs more work....

**Review by chris-calhoun — COMMENTED** [review]
10/10 PERFECT!!!

**Review by chris-calhoun — APPROVED** [review]

**Comment by chris-calhoun** [comment]
my bad

**Comment by drteresavasquez** [comment]
@chris-calhoun then approve me!

**Review by AutumnFoto — APPROVED** [review]

**Pull request merged**